executor/insert_common.go

```diff
@@ -564,15 +564,13 @@ func (e *InsertValues) encodeNewRow(row types.DatumRow) ([]byte, error) {
 
 // getKeysNeedCheck gets keys converted from to-be-insert rows to record keys and unique index keys,
 // which need to be checked whether they are duplicate keys.
-func (e *InsertValues) getKeysNeedCheck(rows []types.DatumRow) ([][]keyWithDupError,
-	error) {
-	nUnique := 0
+func (e *InsertValues) getKeysNeedCheck(insertRows []types.DatumRow) ([][]keyWithDupError, error) {
+	numUniqueIndeices := 0
 	for _, v := range e.Table.WritableIndices() {
 		if v.Meta().Unique {
-			nUnique++
+			numUniqueIndeices++
```

Comment:
Use `indexes` or `indices`. I think we can use another pr to do some rename work.